server/router/router.go

```diff
@@ -444,3 +446,23 @@ func (router *router) KVStore() (kvstore.KVStore, error) {
 func (router *router) Cluster() *cluster.Cluster {
 	return router.cluster
 }
+
+func (router *router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	if req.Method != http.MethodGet {
+		http.Error(w, `{"error": Error method not allowed.Only HTTP GET is accepted}`, http.StatusMethodNotAllowed)
+		return
+	}
+	encoder := json.NewEncoder(w)
```

Comment:
You can use `json.Marshal` which is one line instead of 2 when you don;t need to reuse the encoder.